app/presenters/tree_builder_services.rb

```diff
@@ -18,16 +20,12 @@ def set_locals_for_render
 
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
-    roots = rbac_filtered_objects(Service.roots)
-
-    # Preload the root service picture since it's called by TreeNodeBuilder#build
-    MiqPreloader.preload(roots, :picture)
-
-    count_only_or_objects(count_only, roots, "name")
-  end
-
-  def x_get_tree_service_kids(object, count_only)
-    objects = rbac_filtered_objects(object.direct_service_children.select(&:display).sort_by { |o| o.name.downcase })
-    count_only_or_objects(count_only, objects, 'name')
+    all_services = Rbac.filtered(Service.where(:display => true))
```

Comment:
Removed here:  https://github.com/ManageIQ/manageiq/pull/10098

Agreed, so this is a non-issue.  👍